docs/client/concepts/parameter-stores.mdx

```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
+
+Unlike Feature Gates, Experiments, or Dynamic Configs tied to specific workflows, Parameter Stores offer the flexibility to define and manage parameters independently, which can be remapped to these features later.
+
+---
+
+### **How to Use Parameter Stores: A Practical Example**
+
+Let’s say you want to implement a new onboarding flow, but only for users on the latest version of your app. Here’s how Parameter Stores help without needing constant code updates:
+
+1. **Create a Boolean Parameter**
+
+   In the Statsig Console, create a boolean parameter called `enable_new_onboarding` and set its default value to `false`. This will be your feature toggle.
+
+2. **Use the Parameter in Your Code**
+
+   Fetch the value of the parameter in your app using the Statsig SDK:
+
+   ```java
+   boolean newOnboardingEnabled = StatsigClient.getConfig("enable_new_onboarding").getBoolean("value", false);
+   if (newOnboardingEnabled) showNewOnboardingFlow();
+   else showOldOnboardingFlow();
+   ```
+
+   The app reacts dynamically based on whether the new onboarding flow is enabled, without hardcoding the condition.
+
+3. **Map the Parameter to a Feature Gate**
+
+   Once you're ready to roll out the new onboarding flow to a specific user segment, remap the `enable_new_onboarding` parameter to a Feature Gate that targets users on the latest version.
+
+4. **Control Updates in Real-Time**
 
-- A boolean parameter to control access to a new feature.  Even though you may be used to using Feature Gates for boolean based feature management, start with a boolean parameter instead
-- A string parameter for a text resource that you may want to swap or experiment with
-- A number parameter that you use as input to an onboarding flow (onboarding_steps), a list length to truncate at, etc
-... and many more
+   When you're ready to release the new flow to more users, simply update the parameter in the Statsig Console—no need to release a new app version.
+
+---
+
+### **Best Practices for Using Parameter Stores**
+
+- **Start Small**: Begin with a simple, non-critical feature like `show_promo_banner` and create a boolean parameter to control it remotely.
+- **Experiment with Dynamic Configs**: Remap parameters to Dynamic Configs when you need real-time changes based on user behavior or app version.
+- **Stay Organized**: Group related parameters (e.g., all settings for onboarding) into a single Parameter Store. This simplifies management as your app scales.
+
+---
+
+### **Code Example: Feature Toggle**
+
+Here’s a quick example for Android:
+
+```java
+StatsigClient.initialize(context, "client-sdk-key", statsigUser, () -> {
+    boolean featureEnabled = StatsigClient.getFeatureGate("enable_new_feature").isEnabled();
+    if (featureEnabled) showNewFeature();
+    else showOldFeature();
```

Comment:
the code is wrong

just remove this section. We will make a guided tutorial separately